example/components/Header/Header.tsx

```diff
@@ -5,6 +5,10 @@ import Image from 'next/image';
 const Header = () => {
   return (
     <header className="flex justify-between py-6">
+      <Link  rel="preconnect" href="https://fonts.googleapis.com"></Link>
```

Comment:
The `<Link />` tag from chackra UI is nothing more than a `<a>`. I believe this is to import some external files, so this should be inside the `<head>` tag of the document.

This might be usefull -> https://nextjs.org/docs/pages/building-your-application/optimizing/fonts